stories/atoms/components/listItem/ListItem.stories.mdx

```diff
@@ -123,13 +135,200 @@ The `description` property allows you to provide additional information. It can
   </Story>
 </Canvas>
 
-## Right and left elements
+## Grid layout
+
+If used within a List component the style of the listItem can be changed to a grid style.
```

Comment:
I'm not sure I understand what you mean in this description...

Reply:
If the component (`ListItem`) is a child of `List` then the style of the `ListItem` risks being changed depending on the `List` layout. It's important to be aware since otherwise you might forget to add a style for the grid layout.

Reply:
@ErikssonJoakim I think you should reword a bit this sentence, because, if @lolottetheclash had some difficulties to clearly understand it, let's imagine a lambda user / developer ...